### PR TITLE
Fix: gave right var as class label for confusion matrix

### PR DESCRIPTION
**Pull request by tiaguinho-code**
Was looking into the training script to get how it works and found this mistake. 
**Please do not access the variables in the dataloader directly and only use that for which there is a function.**
This label list for example is sorted based on the input but the y_test and y_train you get as output are sorted based on whatever the binarizer does, so you also get the labels from the binarizer.

### CODE REVIEW

**Review by mlederbauer — APPROVED** [review]
Thank you for the addition! Do you perhaps have an example to show what goes wrong when usng the dataloader variable, compared to y_target? To help me understand a bit

**Comment by tiaguinho-code** [comment]
dataloader.target_unique labels would be this: `[['tertbutoxy3F', 'tertbutoxy9F', 'ethyl', 'thiophenoxy', 'neopentyl', 'Br', 'phenoxy', 'methyl', 'Cl', 'tertbutoxy6F', 'tertbutoxy']]`
and y_labels, that is returned from the dl.load_data() funciton is this: `['Br', 'Cl', 'ethyl', 'methyl', 'neopentyl', 'phenoxy', 'tertbutoxy', 'tertbutoxy3F', 'tertbutoxy6F', 'tertbutoxy9F', 'thiophenoxy']`

Only the one on the bottom is sorted according to the y_test and y_train that is returned. Top one is effectively random.

To take care of this stuff I think I'll go ahead and privatize dataloader variables so you guys can only use that, which is returned by functions and therefore meant to be used.

**Comment by tiaguinho-code** [comment]
Looks like python doesn't have privatization like c++ :(
